plugins/modules/keycloak_client_rolemapping.py

```diff
@@ -144,6 +171,24 @@
         id: role_id2
   delegate_to: localhost
 
+- name: Map a client role to a subgroup, authentication with token
```

Comment:
This is a copy of YAML above with one additional `parents` parameter and slightly changed `name`